### ISSUE TITLE
[BUG] Unable to stop or remove nextcloud stack

### ISSUE THREAD

**Issue by ChaosBlades** [SelfhostedPro/selfhosted_templates#248]
**Describe the bug**
I receive and error message if I try to stop or delete the nextcloud stack. My nextcloud instance is up and working correctly. It does not matter if I leave the containers running, stop them, or delete them. I get the same message.

**To Reproduce**
1. Go to Stacks
2. Click on nextcloud
3. Click on 'Stop this stack' or 'Delete this stack'

**Expected behavior**
The stack to be stopped / deleted

**Error Message**
```
Failure
failed to remove a stack: The PUID variable is not set. Defaulting to a blank string. The PGID variable is not set. Defaulting to a blank string. The TZ variable is not set. Defaulting to a blank string. The PORT variable is not set. Defaulting to a blank string. The MYSQL_ROOT_PASSWORD variable is not set. Defaulting to a blank string. The Timezone variable is not set. Defaulting to a blank string. The DATABASE_PASSWORD variable is not set. Defaulting to a blank string. The Compose file './Template/Stack/nextcloud.yml' is invalid because: services.nextcloud.ports contains an invalid type, it should be a number, or an object : exit status 1
```
All of these env variables are in fact not blank. I also tried to remove the env variables from the stack, save, then delete and still same error message.

**Comment by mbaiti** [comment]
Getting the a similar Error Message when i try to deploy the stack:

> Deployment error
failed to deploy a stack: The Compose file '/data/compose/2/Template/Stack/nextcloud.yml' is invalid because: services.nextcloud.ports contains an invalid type, it should be a number, or an object : exit status 1

**Comment by barthmania** [comment]
I have the exact same problem, did you find a way to delete the stack please ?

**Comment by ChaosBlades** [comment]
Nope. I just discovered this trying to get something else to work so luckily I still need it so it is not too much of a headache. In the future I'll probably just manually add things instead of relying on templates. Does not seem like they are generally maintained very well.
